obspy/core/event/base.py

```diff
@@ -710,6 +796,18 @@ def copy(self):
         return deepcopy(self)
 
     @property
+    def object_id(self):
```

Comment:
Can you add a leading underscore and designate it as private? I see no use to have this as a public part of the API and it frees us to make chances to it in the future if needed or desired.